src/devices/Uln2003/Uln2003.cs

```diff
@@ -136,7 +140,7 @@ public void Step(int steps)
             _stopwatch.Restart();
             _isClockwise = steps >= 0;
             _steps = Math.Abs(steps);
-            _stepMicrosecondsDelay = RPM > 0 ? 60 * 1000 * 1000 / _steps / RPM : StepperMotorDefaultDelay;
+            _stepMicrosecondsDelay = RPM > 0 ? 60 * 1000 * 1000 / _stepsToRotate / RPM : StepperMotorDefaultDelay;
```

Comment:
>_steps [](start = 66, length = 6)

Do we also have to update three lines bellow the while loop to use _stepsToRotate instead of _steps? Do we still need _steps at all?

Reply:
No for first question.
Yes for second. We need _steps to be able to move the motor for fraction of rotation.
If we use _stepsToRotate in while loop inside Step, we would end up with method to always rotate once.
Maybe we can add a new method Rotate:
```csharp
public void Rotate(int rotations)
        {
            Step(rotations * _stepsToRotate); // If the rotations is negative, the motor moves in the reverse direction
        }
```
Thanks to that, people who want to use step motor for multiple rotations will have an easier task.
I and probably some other people want to use step motor to precise alignment, so we need Step with _steps in while loop.